.github/workflows/cla.yml

```
@@ -17,6 +17,31 @@ jobs:
+      - name: Build author-only allowlist
+        id: allowlist
+        if: >
+          github.event_name == 'pull_request_target' ||
+          github.event.comment.body == 'recheck' ||
+          github.event.comment.body == 'I have read and agree to the Contributor License Agreement'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_NUMBER: ${{ github.event.pull_request.number || github.event.issue.number }}
+          PR_AUTHOR: ${{ github.event.pull_request.user.login || github.event.issue.user.login }}
+          BASE_ALLOWLIST: actions-user,ampagent,claude,comfy-pr-bot,github-actions,github-actions[bot]
+,*[bot],Glary Bot
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected alphabetic or numeric character

(syntax-check)